CMakeLists.txt

```diff
@@ -118,3 +118,16 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 # Enable unit testing by including the `test` subdirectory.
 enable_testing()
 add_subdirectory(test)
+
+###############
+## UNINSTALL ##
+###############
+
+# Add uninstall target
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_uninstall.cmake.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/cmake/cmake_uninstall.cmake"
+  IMMEDIATE @ONLY)
+add_custom_target(uninstall
+  "${CMAKE_COMMAND}" -P
+  "${CMAKE_CURRENT_BINARY_DIR}/cmake/cmake_uninstall.cmake")
```

Comment:
Hm, suppose we have `cmake_uninstall.cmake` installed, how could we run the script if the build directory is not present?